vidify/player/web/__init__.py

```diff
@@ -0,0 +1,122 @@
+"""
```

Comment:
You forgot the main dosctring here. Just a brief description of what this player is should be enough.